### PR TITLE
Initial Rendered, (Low) Quality Documentation

### PR DESCRIPTION
**Pull request by dstarner**
https://elastic-hypatia-54e769.netlify.app/#/

This is an initial setup of using [Docsify](docsify.js.org/) to render this documentation repository. It would be fruitful to create a netlify site per pull request once the initial contents are merged, but for now, please refer to the following images as reference as to what it would look like.

The documentation contains the following:

* (AUTO) Full rich markdown support with the following plugins included:
  * `:tag:` emoji support
  * automatic sidebar generation
  * pretty layout and rendering
  * [mermaid graph](docsify.js.org/) functionality
  * "Edit on GitHub" buttons for each page
  * callouts for important information
  * pagination
  * Gitter integration on each page linking to our main (or probably public) chatroom
* (AUTO) search functionality
* a basic `CONTRIBUTING.md` guide that can be linked across projects. The thought being that each project has its own `CONTRIBUTING.md` that is simply a link to this single rendered guide
* basic project background, overviews tthat need to be filled in
* basic development guides and workflows (written as I figure them out)
* TODO: service workers for offline availability (i have the code, i just dont want to integrate until we have something worthwhile)

Screen Shot 2020-06-15 at 12 11 30 AM
Screen Shot 2020-06-15 at 12 11 45 AM
Screen Shot 2020-06-15 at 12 11 39 AM
Screen Shot 2020-06-15 at 12 12 26 AM

### CODE REVIEW

**Comment by JamesWP** [comment]
Wow looks great

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+### Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
+complaints will be reviewed and investigated and will result in a response that
```
Comment (JamesWP) [inline CONTRIBUTING.md:104]: Insert email address?? Might want to think about what to put here

**Review by JamesWP — APPROVED** [review]
This is nice, let’s get it up somewhere and ask someone from the charity itself if this is ok

File: CONTRIBUTING.md (hunk 1 above)
Comment (dstarner) [reply CONTRIBUTING.md:104]: I can drop the gitter link instead of the email, since there isnt really one unique email @JamesWP

**Pull request merged**